CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
-# dbt_apple_store_source v0.UPDATE.UPDATE
-
- ## Under the Hood:
-
+# dbt_apple_store_source v0.3.0
+## Under the Hood
+[PR #9](https://github.com/fivetran/dbt_apple_store_source/pull/9) includes the following changes:
 - Incorporated the new `fivetran_utils.drop_schemas_automation` macro into the end of each Buildkite integration test job.
 - Updated the pull request [templates](/.github).
```

Comment:
Purely stylist can you move the Under the Hood section to be after the Bug Fixes section.